x/rollappparams/types/params.go

```diff
@@ -0,0 +1,138 @@
+package types
+
+import (
+	"fmt"
+	"regexp"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/dymensionxyz/dymint/da/registry"
+)
+
+const (
+	DefaultDA           = "celestia"
+	DefaultCommit       = "74fad6a00713cba62352c2451c6b7ab73571c515"
+	DefaultBlockMaxGas  = 400000000
+	DefaultBlockMaxSize = 500000
+	MinBlockMaxSize     = 100000
+	MinBlockMaxGas      = 10000000
+	CommitLength        = 40
+)
```

Comment:
can you comment how these values were arrived at

Reply:
added